docker/dockerfiles/Dockerfile.node-16.14-ember

```
@@ -0,0 +1,67 @@
+FROM node:16.14.0
+# ember server on port 4200
+# livereload server on port 7020 (changed in v2.17.0 from 49153)
+# test server on port 7357
+EXPOSE 4200 7020 7357
+WORKDIR /code
+
+
+# run ember server on container start
+
+# Install watchman build dependencies 
+RUN \
+	apt-get update -y &&\
+	apt-get install -y python-dev
```

```suggestion
	apt-get install -y --no-install-recommends python-dev
```

C'est mieux d'utiliser cette option lors de la construction d'une image pour maitriser le volume de celle-ci

---

```suggestion
	apt-get install -y python-dev &&\
	apt-get clean &&\
  rm -rf /var/lib/apt/lists/*
```
Après l'installation de paquets, pas mal de petits documents inutiles au RUN du conteneur sont créés dans les dossiers d'APT, Docker recommande un nettoyage de ces fichiers : https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

```
In addition, when you clean up the apt cache by removing /var/lib/apt/lists it reduces the image size, since the apt cache is not stored in a layer.
```

---

Plus d'apt-get install donc plus de problèmes